Polyhedron/demo/Polyhedron/MainWindow.cpp

```diff
@@ -2304,11 +2306,6 @@ void MainWindow::on_actionPreferences_triggered()
     this->s_defaultPSRM = CGAL::Three::Three::modeFromName(text);
   });
 
-  connect(prefdiag.backFrontColor_pushButton, &QPushButton::clicked,
-          this, [](){
-    qobject_cast<Viewer*>(CGAL::Three::Three::activeViewer())->setBackFrontColors();
-  });
-
```

Comment:
What happened to that code?

Reply:
A rebasing error. I mixed it up with setBackFrontShading.